src/Libraries/Microsoft.Extensions.AI.Abstractions/TextToImage/TextToImageOptions.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Drawing;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.Extensions.AI;
+
+/// <summary>Represents the options for an image generation request.</summary>
+[Experimental("MEAI001")]
+public class TextToImageOptions
+{
+    /// <summary>Gets or sets the content type of the image.</summary>
+    public TextToImageContentType? ContentType { get; set; }
+
+    /// <summary>
+    /// Gets or sets the number of images to generate.
+    /// </summary>
+    public int? Count { get; set; }
+
+    /// <summary>
+    /// Gets or sets the guidance scale to use for image generation.
+    /// </summary>
+    public float? GuidanceScale { get; set; }
```

Comment:
I had to look up what this means. Is this the accepted vernacular for this concept? It's fine if it is, just confirming.

Reply:
It is, but as above there is just one model that accepts this.  I'm inclined to remove it.